trafaret/numeric.py

```diff
@@ -187,8 +187,15 @@ def check_and_return(self, data):
 
 
 class ToDecimal(Float):
+    """
+    Checks that value is a Decimal.
+    Or if value is a string converts this string to Decimal.
+    """
     value_type = decimal.Decimal
 
+    def check_and_return(self, data):
+        return self._check(data)
+
```

Comment:
why did u rewrite this method?

Reply:
to make `ToDecimal` return `Decimal`, not raw data. The same way `ToInt` & `ToFloat` do.

Reply:
@Deepwalker do we need `Decimal` or this is the same with `Float`?